src/SqlPersistence.Tests/Saga/SagaPersisterTests.cs

```diff
@@ -47,8 +47,7 @@ SagaPersister SetUp(string endpointName)
             readerCreator: reader => new JsonTextReader(reader),
             writerCreator: writer => new JsonTextWriter(writer),
             tablePrefix: $"{endpointName}_",
-            schema: schema,
-            sqlVariant: runtimeSqlVariant,
+            sqlDialect: runtimeSqlVariant,
```

Comment:
shouldnt we move all "Variant" over to "Dialect? so future people dont need to mind map on variable/param/method names?